pyproject.toml

```diff
@@ -262,7 +262,7 @@ version = "~3.2.0"
 optional = true
 
 [tool.poetry.dependencies.transformers]
-version = "~4.13.0"
+version = ">=4.13.0, <=4.26.0"
```

Comment:
Based on this [comment](https://rasahq.atlassian.net/browse/ENG-85?focusedCommentId=16951), this should be a strict `<4.26.0` rather.

Reply:
Thomas suggested me to keep it `<=4.26.0`. And I also verified that nothing breaks in `4.26.0`

Reply:
Sounds good, just thought of double-checking.